gulpfile.js

```diff
@@ -197,6 +203,37 @@ async function addDistToIndex() {
 
 const cliVersion = '1.8.5';
 
+async function nugetInstallPortalPackages() {
+    const downloadDir = path.resolve(outdir, 'temp_portal_package', 'download');
+    const unzpiDir = path.resolve(outdir, 'temp_portal_package', 'unzipped_package');
+    const packageName = "CDSStarterPortal"
+    const packageNameToImport = `Adxstudio.${packageName}`
+    const portalPackageOutDir = path.resolve(outdir , 'portal_package')
+
+    await nugetInstall('PowerPortalPackages', 'cdsstarterportal', '9.2.2103.21', downloadDir );
+
+    log.info(`Extracting package zip into folder: ${unzpiDir}`);
+    return new Promise((resolve, reject) => {
+        fs.createReadStream(path.resolve(downloadDir, `${packageName}.zip`)).pipe(unzip.Extract({ path: unzpiDir }))
+            .on('close', () => {
+                fs.createReadStream(path.resolve(unzpiDir , `${packageNameToImport}.zip`)).pipe(unzip.Extract({ path: portalPackageOutDir}))
```

Comment:
since line 217 already creates a pipe, you should be able to pipe through the inner unzip?

Reply:
Done